apps/src/dev/app.html.ejs

```diff
@@ -47,7 +47,7 @@
       console.log(this);
     };
 
-    var appOptions = <%-: options | json %> ;
+    var appOptions = <%- JSON.stringify(options) %> ;
```

Comment:
[ejs](https://github.com/mde/ejs) >= 2.x dropped support for built-in filters, so I've replaced the few references in our codebase to the `: | json` filter with the more verbose JSON.stringify.